docs/Installation/Installation_Guide.rst

```diff
@@ -284,15 +270,15 @@ credentials to access the cluster. Export this location in the shell to give
 the :program:`kubectl` and :program:`helm` tools the correct paths and
 credentials to contact the cluster *kube-master* nodes::
 
-  (metal-k8s) $ export KUBECONFIG=`pwd`/inventory/quickstart-cluster/artifacts/admin.conf
+  (metalk8s) $ export KUBECONFIG=`pwd`/inventory/quickstart-cluster/artifacts/admin.conf
 
 If your system can reach port *6443* on the first *kube-master* node, you can
 
 * List the nodes
 
-::
+  ::
 
-   (metal-k8s) $ kubectl get nodes
+   (metalk8s) $ kubectl get nodes
    NAME        STATUS    ROLES            AGE       VERSION

```

Comment:
Since the example `hosts` file has 5 nodes, this example should also list 5 nodes for consistency.

Reply:
done.

Reply:
Now that the example `hosts` file has been changed to 3 servers, please revert the change here  so the document is consistent.